server/schedulers/hot_region.go

```diff
@@ -95,11 +95,13 @@ type hotScheduler struct {
 	r           *rand.Rand
 
 	// states across multiple `Schedule` calls
-	pendings       [resourceTypeLen]map[*pendingInfluence]struct{}
+	pendings [resourceTypeLen]map[*pendingInfluence]struct{}
+	// regionPendings stores regionID -> [opType]Operator
```

Comment:
I think a better comment is to explain how these variables work.

Reply:
updated.